packages/cli/lib/prompts/accountsPrompt.js

```diff
@@ -1,7 +1,19 @@
 const { updateDefaultAccount } = require('@hubspot/cli-lib/lib/config');
 const { promptUser } = require('./promptUtils');
 const { i18n } = require('@hubspot/cli-lib/lib/lang');
-const { mapAccountChoices } = require('./sandboxesPrompt');
+
+const getSandboxType = type =>
+  type === 'DEVELOPER' ? 'development' : 'standard';
+
+const mapAccountChoices = portals =>
+  portals.map(p => {
+    const isSandbox = p.sandboxAccountType && p.sandboxAccountType !== null;
+    const sandboxName = `[${getSandboxType(p.sandboxAccountType)} sandbox] `;
+    return {
+      name: `${p.name} ${isSandbox ? sandboxName : ''}(${p.portalId})`,
+      value: p.name || p.portalId,
+    };
+  });
```

Comment:
Separated out the account mapping for the accounts prompt, and added in an undefined check for sandboxAccountType to support older configs